doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-global-settings-panel-rule-based-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='test-designer-global-settings-panel-rule-based-proc']
+= Global settings for rule-based test scenarios
+
+Follow the procedure below to view and edit the global settings of the rule-based test scenarios.
+
+.Procedure
+. Click `Settings` tab on the right side of the test scenario designer to display the attributes.
+. Configure the following attributes in the `Settings` panel.
+* `Name`: You can change the name of the existing test scenarios by using the by using the `Rename` option from the upper-right toolbar in the designer.
+* `Type`: This attribute specifies it is a rule-based test scenario and it is read-only.
+* `KieSession`: (Optinal) Enter the KieSession for the test scenario.
+* `RuleFlowGroup/AgendaGroup`: (Optinal) Enter the RuleFlowGroup or AgendaGroup for the test scenario.
+. To skip the simulation during the test, select the check box.
```

Comment:
Suggestion - add "Optionally" to step:

s/To skip the simulation during the test, select the check box./ Optionally, to skip the simulation during the test, select the check box.